app/controllers/topics_controller.rb

```diff
@@ -970,7 +970,12 @@ def reset_new
         end
       end
 
-    dismissed_topic_ids = DismissTopics.new(current_user, topic_scope).perform!
+    if params[:topic_ids].present?
+      topic_ids = params[:topic_ids].map { |t| t.to_i }
```

Comment:
will this end up with a 500 error if you do `topic_ids=1` ? do we want to ensure it is an array?